scripts/test/mender-convert-qemu

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "$DISK_IMG" ]; then
+    echo "ERROR: DISK_IMG is not set"
+    exit 1
+fi
+
+# Try common locations for OVMF
+ovmf_file=""
+for maybe_ovmf_file in /usr/share/OVMF/OVMF_CODE.fd /usr/share/ovmf/x64/OVMF_CODE.fd; do
```

Comment:
minor: `find /usr -type f -and -name OVMF_CODE.fd | head -1` in case we need to really look for it.

Reply:
Ok, thanks!